src/cls/queue/cls_queue_ops.h

```diff
@@ -136,4 +136,42 @@ struct cls_queue_get_capacity_ret {
 };
 WRITE_CLASS_ENCODER(cls_queue_get_capacity_ret)
 
+struct cls_queue_get_size_ret {
+  uint64_t queue_size;
+
+  cls_queue_get_size_ret() {}
+
+  void encode(ceph::buffer::list& bl) const {
+    ENCODE_START(1, 1, bl);
+      encode(queue_size, bl);
+    ENCODE_FINISH(bl);
+  }
+
+  void decode(ceph::buffer::list::const_iterator& bl) {
+    DECODE_START(1, bl);
+      decode(queue_size, bl);
+    DECODE_FINISH(bl);
+  }
+};
+WRITE_CLASS_ENCODER(cls_queue_get_size_ret)
+
+struct cls_queue_get_committed_entries_ret {
```

Comment:
can we merge `cls_queue_get_size_ret` and `cls_queue_get_committed_entries_ret` to one struct + API?

Reply:
yes we can